src/apps/service/apps/jobs/util.clj

```diff
@@ -51,8 +52,7 @@
    "FolderInput"       "collection"
    "MultiFileSelector" "many"})
 
-(def input-types
-  (set (keys input-multiplicities)))
+(def input-types am/param-ds-input-types)
```

Comment:
The HPC `FileFolderInput` type was missing from this before, so inputs were not repopulating in the UI relaunch forms correctly.